### PR TITLE
[Bugfix:Developer] Fix flakyness of Cypress tests

### PR DESCRIPTION
**Pull request by Youssef-Chip**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant
* [ ] Screenshots are attached to Github PR if visual/UI changes were made

### What is the current behavior?
There are 3 current bugs with cypress tests: 
Flakyness in edit-gradeable.spec.js
Removing screenrecorded videos when no tests where being run
Flakyness in Docker_ui.spec.js
### What is the new behavior?
This PR fixes the first 2 bugs.

### CODE REVIEW

**Comment by williamjallen** [comment]
If you have fixes for two separate issues, the fixes should be made as two separate PRs.

File: site/cypress/e2e/Cypress-Gradeable/edit_gradeable.spec.js [hunk 1]
```diff
@@ -151,7 +151,9 @@ describe('Tests cases revolving around modifying gradeables', () => {
         cy.get('#add-peer-grader').click();
         cy.get('.form-button-container > a').click({ force: true, multiple: true });
 
-        cy.get('#page_5_nav').click();
+        //clicking on page 5 directly, loads the page too fast, which causes flakyness
+        //so we better visit the url
+        cy.visit(['sample', 'gradeable', 'open_peer_homework', 'update?nav_tab=5']);
```
Comment (williamjallen) [inline site/cypress/e2e/Cypress-Gradeable/edit_gradeable.spec.js:156]: I believe you could still test the link by first asserting that the link is visible, and then clicking on it.

**Comment by Youssef-Chip** [comment]
> If you have fixes for two separate issues, the fixes should be made as two separate PRs.

The problem is The tests would still fail, so I want a PR that passes all the test

File: site/cypress/e2e/Cypress-Gradeable/edit_gradeable.spec.js (hunk 1 above)
Comment (Youssef-Chip) [reply site/cypress/e2e/Cypress-Gradeable/edit_gradeable.spec.js:156]: doesn't cy.visit already do that by default?

Comment (IDzyre) [reply site/cypress/e2e/Cypress-Gradeable/edit_gradeable.spec.js:156]: Cy.visit() doesn't check the links that are on the page, it just tries to visit a URL.

**Comment by williamjallen** [comment]
> The problem is The tests would still fail, so I want a PR that passes all the test

The flaky tests being addressed here are in two separate test suites though, so it should be clear whether each fix works independently.  We just don't want to block one fix if the other fix runs into a snag during the review process (something like the above comment, for example).

File: site/cypress/e2e/Cypress-Gradeable/edit_gradeable.spec.js (hunk 1 above)
Comment (Youssef-Chip) [reply site/cypress/e2e/Cypress-Gradeable/edit_gradeable.spec.js:156]: I tried to put ```cy.get('#page_5_nav').should('be.visible')```  followed by ```cy.get('#page_5_nav').click()``` or         ```cy.get('#page_5_nav').as('page5');  cy.get('@page5').click();``` but it is still flaky

**Pull request merged**